LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 Kentico
+Copyright (c) 2022 Kontent.ai s.r.o.
```

Comment:
I think that here should be Kontent s.r.o. as it is a legal name of company (also management sdk also has it :D )

Reply:
good catch, will update